tracing-example/src/main/java/org/jboss/resteasy/tracing/examples/jackson/Play.java

```diff
@@ -0,0 +1,21 @@
+package org.jboss.resteasy.tracing.examples.jackson;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.lang.reflect.Method;
+
+public class Play {
+    public static void main(String[] args) throws Exception {
```

Comment:
What's this class for? I don't see it mentioned in the doc nor in the code...

Reply:
discarded